packages/cloudflare/src/cli/build/patches/plugins/route-module.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Patch for `next/src/server/route-modules/route-module.ts`
+ * https://github.com/vercel/next.js/blob/c8c9bef3a1dd1a95434c1865dbd7f69985ea7d54/packages/next/src/server/route-modules/route-module.ts#L389-L437
```

Comment:
That should be enough
```suggestion
 * https://github.com/vercel/next.js/blob/c8c9bef/packages/next/src/server/route-modules/route-module.ts#L389-L437
```